models/desired_lrp_test.go

```diff
@@ -122,7 +123,8 @@ var _ = Describe("DesiredLRP", func() {
                 "nofile": 16384
               },
               "user": "vcap",
-			  "suppress_log_output": false
+			  "suppress_log_output": false,
+              "volume_mounted_files": null
```

Comment:
I see that this tabbing was weird before you were in the file, but can you fix it since you are in this file?